NeuralNetwork/save_system.cpp

```diff
@@ -77,7 +78,7 @@ void load_network(Network & network, std::string file_name)
 			layer++;
 			break;
 		default:
-			_ASSERT(true);
+			assert(false);
```

Comment:
`_ASSERT` is probably a windows thing, never heard of it. The standard one is `assert`, in the header `<cassert>`. Also, I think you mean false (assertions fail if they're false, not true)